src/components/Registers.js

```diff
@@ -23,11 +36,11 @@ const RegisterPanel = ({ registers }) => {
   );
 };
 
-RegisterPanel.propTypes = {
-  registers: PropTypes.arrayOf(PropTypes.shape({
-    low_: PropTypes.number,
-    high_: PropTypes.number,
-  })).isRequired,
-};
+// RegisterPanel.propTypes = {
```

Comment:
how come we got rid of theses?

Reply:
it doesnt take a prop anymore, so I think its unnecessary

Reply:
that's true, feel free to remove the commented out code